queries/java/highlights.scm

```diff
@@ -90,6 +90,12 @@
 (constructor_declaration
   name: (identifier) @type)
 (type_identifier) @type
+((method_invocation
+  object: (identifier) @type)
+ (#lua-match? @type "^[A-Z]"))
```

Comment:
Can you help me with that? Do you want to match `Foo::bar()`? In this case you should rather include the `::` instead of doing a regex or do you want to highlight also `Foo.bar()`

Reply:
My fault, `::` is only used in Java when you need the function for a higher-order function.